### PR TITLE
fix #2601

### LINKED ISSUE #2601: color code legend for visual job tracking: feature request
in the point-and-click interface of queueing up an analysis, it would be helpful to have a little legend with a color code that explains what the different color circles mean.  thanks!

https://qiita.ucsd.edu/analysis/description/17522/

### PR DESCRIPTION
**Pull request by antgonza**
The new visualization is:
screen shot 2018-07-05 at 8 06 46 am

### CODE REVIEW

**Comment by antgonza** [comment]
Good question! (... and not sure). I see two options:
1. ignore those 2 elements
2. add more information for each of them

Both options means special cases (if/else) in the for loop so no preference there but I I like 1 more, thoughts?

**Comment by ElDeveloper** [comment]
Ignoring those would probably be confusing since the gray color of type and queued/waiting are so similar. Perhaps a better idea is to divide the legend in two rows, one for statuses and another one for types?

**Comment by antgonza** [comment]
New version:
screen shot 2018-07-05 at 12 37 11 pm

File: qiita_pet/static/js/networkVue.js [hunk 1]
```diff
@@ -1087,6 +1088,29 @@ Vue.component('processing-graph', {
       vm.update_job_status();
     });
 
+    var circle_statuses = [];
+    var circle_types = [];
+    for (circle_name in vm.colorScheme) {
```
Comment (ElDeveloper) [inline qiita_pet/static/js/networkVue.js]: `circle_name` -> `var circle_name`

**Pull request merged**